pylhc/machine_settings_info.py

```diff
@@ -154,77 +175,89 @@ def log_summary(acc_time: AccDatetime, bp_info: DotDict, o_info: DotDict, trims:
     """Log the summary."""
```

Comment:
I would like to see a docstring here for the different arguments.

Reply:
Also maybe default the arguments to `None` as this is the performed check later on